scripts/mapo

```diff
@@ -191,6 +199,72 @@ def register():
     register_all_environments()
 
 
+def make_config(args):
+    """Make config dict to trainer."""
+
+    # === Environment ===
+    env_config = {"env": tune.grid_search(args.env)}
+
+    # === MAPO ===
+    mapo_config = {}
+    if args.run in ["MAPO", "OffMAPO"]:
+        mapo_config = {
+            "branching_factor": args.branching_factor,
+            "use_true_dynamics": args.use_true_dynamics,
+            "model_loss": args.model_loss,
+        }
+
+    # === Models ===
+    models_config = {
+        "model": {
+            "custom_model": "mapo_model",
+            "custom_options": {
+                "actor": args.config_actor_net,
+                "critic": args.config_critic_net,
+            },
+        }
+    }
+    if args.run in ["MAPO", "OffMAPO"]:
+        models_config["model"]["custom_model"] = "mapo_model"
+
+        if not args.use_true_dynamics:
+            models_config["model"]["dynamics"] = args.config_dynamics_net
```

Comment:
This should go under "custom_options" as in "actor" and "critic" above.

Reply:
done!